src/gui/tray/Window.qml

```diff
@@ -621,21 +621,19 @@ ApplicationWindow {
                 }
 
                 HeaderButton {
-                    id: trayWindowTalkButton
-
-                    visible: UserModel.currentUser && UserModel.currentUser.serverHasTalk
-                    icon.source: "image://svgimage-custom-color/talk-app.svg" + "/" + Style.currentUserHeaderTextColor
+                    id: trayWindowFeaturedAppButton
+                    visible: UserModel.currentUser.isFeaturedAppEnabled
+                    icon.source: UserModel.currentUser.featuredAppIcon + "/" + Style.currentUserHeaderTextColor

```

Comment:
This is the recolouring I'm referring to

Reply:
If you are going to go with the svg provider recolouring method I would remove the `icon.color` method below